diff --git a/shell/platform/glfw/flutter_glfw.cc b/shell/platform/glfw/flutter_glfw.cc
index 20f2c8617..92904b8ed 100644
--- a/shell/platform/glfw/flutter_glfw.cc
+++ b/shell/platform/glfw/flutter_glfw.cc
@@ -534,7 +534,7 @@ FlutterDesktopWindowControllerRef FlutterDesktopCreateWindow(
 
   // Create the window, and set the state as its user data.
   state->window = UniqueGLFWwindowPtr(
-      glfwCreateWindow(initial_width, initial_height, title, NULL, NULL),
+      glfwCreateWindow(initial_width, initial_height, title, glfwGetPrimaryMonitor(), NULL),
       glfwDestroyWindow);
   GLFWwindow* window = state->window.get();
   if (window == nullptr) {
