diff --git a/source/i18n/plurrule.cpp b/source/i18n/plurrule.cpp
index fd193560..09200fb8 100644
--- a/source/i18n/plurrule.cpp
+++ b/source/i18n/plurrule.cpp
@@ -1661,7 +1661,7 @@ int64_t FixedDecimal::getFractionalDigits(double n, int32_t v) {
       case 3: return (int64_t)(fract*1000.0 + 0.5);
       default:
           double scaled = floor(fract * pow(10.0, (double)v) + 0.5);
-          if (scaled > U_INT64_MAX) {
+          if ((int64_t)scaled > U_INT64_MAX) {
               return U_INT64_MAX;
           } else {
               return (int64_t)scaled;
